src/main/java/org/kainos/ea/resources/SalesEmployeeController.java

```diff
@@ -0,0 +1,43 @@
+package org.kainos.ea.resources;
+
+import io.swagger.annotations.Api;
+import org.kainos.ea.api.SalesEmployeeService;
+import org.kainos.ea.cli.SalesRequest;
+import org.kainos.ea.client.FailedToCreateSalesException;
+import org.kainos.ea.client.InvalidSalesEmployeeException;
+import org.kainos.ea.core.SalesEmployeeValidator;
+import org.kainos.ea.db.SalesDao;
+
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.UriBuilder;
+import java.net.URI;
+
+@Api("Team SCM SalesEmployee API")
+@Path("/api")
+public class SalesEmployeeController {
+
+    private final SalesEmployeeService salesEmployeeService = new SalesEmployeeService(new SalesDao(), new SalesEmployeeValidator());
+
+    @POST
+    @Path("/SalesEmployee")
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response createSalesEmployee(SalesRequest employee){
+        try{
+            salesEmployeeService.createSales(employee);
+            URI uri = UriBuilder.fromPath("/SalesEmployee").build();
```

Comment:
why are you returning a uri here, you should just return the id of the created employee